.github/actions/benchmark-build/benchmark-macOS.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+# This uses bash variable substitution in a few places
+# 1. getting the cmake directory for running cpack with an absolute path (chocolatey has an unfortunately named alias)
+
+brew install swig boost
```

Comment:
Can probably also remove swig from here as well, unless this is used for other things as well.